src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -142,6 +144,14 @@ private async Task<bool> IsEnabledWithVariantsAsync<TContext>(string feature, TC
                 await sessionManager.SetAsync(feature, isFeatureEnabled).ConfigureAwait(false);
             }
 
+            PublishTelemetry(new EvaluationEvent
+            {
+                FeatureDefinition = featureDefinition,
+                TargetingContext = targetingContext,
```

Comment:
targetingContext isn't set if the feature is disabled. Intended?

Reply:
I originally thought yes, as if it wasn't relevant for the evaluation, it shouldn't be included. But that's ignoring the call to IsEnabledAsync, which uses the targeting context. So I'll rework this.